### DISCUSSION TITLE
Support  LightShows

### DISCUSSION THREAD

**Discussion by 0burner0** [marcone/teslausb#863] [Ideas]
I have a feature request related to the TESLAUSB project. I would like to have the ability to create a partition on a standard USB stick specifically for storing and managing lightshows. This would be a convenient way to organize lightshow files separately from other data on the USB stick.

Details:

Current Behavior: TESLAUSB works perfectly for its intended functions, but I couldn't find an option to create a dedicated partition for lightshows.
Requested Feature: Add an option to create a separate partition on the USB stick for lightshow files.
Benefits: This feature would allow users to manage lightshow files more efficiently, keeping them organized and separate from other data.
Conclusion:

I appreciate the excellent functionality of TESLAUSB so far. Adding this feature would make it even more versatile and user-friendly. Thank you for considering this request!

**Comment by marcone** [comment]
> Current Behavior: TESLAUSB works perfectly for its intended functions, but I couldn't find an option to create a dedicated partition for lightshows.

The current version of the config file shows how to specify lightshow and boombox drives:
https://github.com/marcone/teslausb/blob/main-dev/pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample#L97

**Reply by 0burner0** [reply]
Who sorry i used a old configfile.. did not see that option. Thank you verry much!!

**Comment by miles267** [comment]
@marcone Once you've added the EXPORT LIGHTSHOW_SIZE=1G and re-run the teslausb setup (and verified the new volume was created), how do you add light show data to it?  I cannot seem to find instructions for this anywhere.  If you can please link me, it would be appreciated.  Thanks.

**Reply by marcone** [reply]
https://github.com/teslamotors/light-show describes what to put on the lightshow drive.

**Reply by miles267** [reply]
Thank you.  Is the preferred way of uploading the light show data files via the TeslaUSB WebUI menu? Or is there the option to sync from a network share just like you can from a designated music share on NAS?

**Reply by marcone** [reply]
There is no option to sync from a network share, so using the web interface is probably most convenient.
